### ISSUE TITLE
Support `[ and `] to jump to beginning/end of previously changed or yanked text

### ISSUE THREAD

**Issue by angularsen** [jaredpar/VsVim#1151]
``[` and ``]` doesn't seem to do anything in VsVim. It is also not shown when typing `:marks`.
However, the related ``<` and ``>` marks seem to work.

Need this for [Selecting your pasted text](http://vim.wikia.com/wiki/Selecting_your_pasted_text)
"[...] select the text you just pasted. For example, you may want to paste some text, then select it for indenting, or formatting, or performing a substitution."

See special marks section:
http://vim.wikia.com/wiki/Using_marks

**Comment by squerl** [comment]
For anyone else finding this issue and waiting eagerly, I'd like to recommend marks that cover the important use cases of missing **\`] `[**:

**`.** jumps to position last changed in buffer. If you paste a block of text, I used to use **`]** to jump to the end of it, but **`.** does the exact same thing.

**`<** jumps to beginning of last visual selection. When I would highlight a selection and yank and want to jump to the beginning of that, I would use **`[**, but **`<** does the same thing too!

Hope this helps.

**Comment by anjdreas** [comment]
@squerl Thanks for the tip on \`. In my usecase, I just want to format the text I just pasted for indentation etc. So I can now solve this with ```v`.``` or create an alias for it.

Example:
Yank some lines of text
Go somewhere and paste it
Cursor is now at beginning of pasted text
Enter select mode ```v```
Go to end of pasted text ``` `.```
Format text ```=```

**Comment by anjdreas** [comment]
I wound up with these two aliases:

```
" Paste and then format the pasted text
nnoremap fp pv`.= 
" Paste over selection, and then format the pasted text
vnoremap fp "_dPv`.= 
```

**Comment by ricksladkey** [comment]
I have implemented this but I can't submit it until PR #2139 is merged. Here is a screenshot after yanking a visual selection and pasting it:

![image](https://user-images.githubusercontent.com/2680524/41272951-644c1228-6de5-11e8-936f-57013a69f244.png)

**Comment by Stamo-Gochev** [comment]
Nice! When the update will be available for the VS extension?
